pennylane/transforms/dynamic_one_shot.py

```diff
@@ -50,7 +50,7 @@ def null_postprocessing(results):
 @transform
 def dynamic_one_shot(
     tape: qml.tape.QuantumTape, **kwargs
-) -> tuple[Sequence[qml.tape.QuantumTape], Callable]:
+) -> Tuple[Sequence[qml.tape.QuantumTape], Callable]:
```

Comment:
Python 3.9 added support for using standard collections as type hints so we don't need to use `typing.Tuple`. `tuple` should be fine, and I think it might even be better to start standardizing its use in our type hints.

```suggestion
) -> tuple[Sequence[qml.tape.QuantumTape], Callable]:
```